.editorconfig

```diff
@@ -5,3 +5,6 @@ indent_style = space
 indent_size = 4
 end_of_line = lf
 charset = utf-8
+
+[*.md]
+indent_size = 2
```

Comment:
I know it's not really part of the commit, but just make sure that if the current indent size is 4, then this matches to be 4. If it is 2, then keep it 2.

Reply:
The thing is that the readme examples look better with 2 spaces, because it makes it easier to read on mobile. the 2 spaces only apply to .md, the rest follow 4 spaces